index.html

```diff
@@ -1,138 +1,13 @@
 <!DOCTYPE html>
 <html lang="en">
 
+<!-- https://stackoverflow.com/a/27895590 -->
 <head>
-    <meta charset='utf-8'>
-    <meta http-equiv="X-UA-Compatible" content="chrome=1">
-    <meta name="description" content="Reaction Mechanism Generator : RMG organization&#39;s &quot;home&quot; website">
-
-    <link rel="stylesheet" type="text/css" media="screen" href="stylesheets/stylesheet.css">
-
-    <title>Reaction Mechanism Generator</title>
+    <meta http-equiv='refresh' content='0; URL=http://example.com/'>
```

Comment:
target URL?

Reply:
Going to redirect to https://github.com/ReactionMechanismGenerator/RMG-Py just testing for now